test/integration/BalancerV3.t.sol

```diff
@@ -133,7 +133,7 @@ abstract contract BalancerV3Test is SettlerMetaTxnPairTest, AllowanceHolderPairT
     function testBalancerV3() public skipIf(balancerV3Pool() == address(0)) setBalancerV3Block {
         (ISignatureTransfer.PermitTransferFrom memory permit, bytes memory sig) = _getDefaultFromPermit2();
 
-        (uint256 hashMul, uint256 hashMod) = perfectHash();
+        (uint256 hashMul, uint256 hashMod) = BalancerV3Test.perfectHash();
```

Comment:
what made this change necessary?

Reply:
This is because I kept the `perfectHash` with the same name in both EkuboTest and BalancerV3Test, so USDCUSDTTest needs to override `perfectHash` then I needed to use the explicit one I want to use on each, maybe better to just change the names of the functions to avoid the override (which is not needed). I am going to do that